app_usb_aud_xk_216_mc/src/core/xua_conf.h

```diff
@@ -10,7 +10,7 @@
 #include "../../../shared/version.h"
 
 /*
- * Device configuration option defines to override default defines found lib_xua/api/xua_conf_defaults.h
+ * Device configuration option defines to override default defines found in lib_xua/api/xua_conf_defaults.h
```

Comment:
In the comments in all of these files, this filename is actually `xua_conf_default.h`